<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_R">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Tegangan R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_S">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Tegangan S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_T">
          <property name="styleSheet">
           <string notr="true">
font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Tegangan T</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Kondisi">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="line_R">
          <property name="baseSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_S">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_T">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_Status_T">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Arus R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Arus S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Arus T</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size:10pt;</string>
          </property>
          <property name="text">
           <string>Status Arus</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="line_arus_R">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_arus_S">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Line_arus_T">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_status_A">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="topMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="Button_Start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Box_port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Port...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_R" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_S" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="widget_RST" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_T" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>50</verstretch>
           </sizepolicy>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:black;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
